metadata.json

```diff
@@ -29,7 +29,8 @@
       "operatingsystemrelease": [
         "30",
         "31",
-        "32"
+        "32",
```

Comment:
I don't mind these changes, but could you add them in a separate PR for a complete changelog?